site-scrapers/Hannaford.js

```diff
@@ -12,10 +12,9 @@ module.exports = async function GetAvailableAppointments(browser) {
         const response = webData[loc.zip];
         return {
             name: `${siteName} (${loc.city})`,
-            hasAvailability: false, // with current impl of rxTouch, we can't determine for sure if there's availability yet 
-            extraData: response && response.length
-                ? response.substring(1, response.length - 1) //take out extra quotes
-                : response,
+            hasAvailability: Object.keys(response.availability).length != 0 ? true : false,
```

Comment:
!==

Reply:
Actually any time you write a ternary like `case ? true : false` you know you just need to write `case` :)
so,
```
Object.keys(response.availability).length !== 0
```
and that's it.

Reply:
alternatively you can do `!!Object.keys(response.availability).length` which I like

Reply:
done!